src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
I will solve at least those two things you pointed out already. I still have no idea how to solve the problem I mentioned earlier though (how to get the output from the script to the dosemu window). If you have a clue what I'm doing wrong there, I'd really appreciate if you could give me a pointer :)